lib/mailtemplate.php

```diff
@@ -124,8 +124,10 @@ public function reset(){
 					if (rename($absolutePath.'.bak', $absolutePath)) {
 						return true;
 					}
-				} else if (unlink($absolutePath)) {
-					return true;
+				} else if(file_exists($absolutePath)) {
+					if (unlink($absolutePath)) {
```

Comment:
I was against this, because then the logic makes more sense. Just to have it grouped logically - "only unlink if the file_exists" and "when the unlink succeeds then return success". 


Sure from an logical point this can be combined but just makes things harder to understand in the future.